app/components/ui/GDocEmbed/GDocEmbed.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Iframe from 'react-iframe';
+import Footer from '../Footer/Footer';
+
+import styles from './GDocEmbed.scss';
+
+
+export default class GDocEmbed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Defining an empty state is a bit misleading and unnecessary. If the component is stateless, then you don't need to declare this field. In fact, I would prefer that stateless components are declared as [function components](https://reactjs.org/docs/components-and-props.html#function-and-class-components), which are much simpler.

In this case, you can completely drop the `class` ceremony and define a function that is just the body of the `render()` method and call it a day. It'd look something like:

```jsx
const GDocEmbed = ({ pageHeader, GDocURL }) => (
  <React.Fragment>
    <article className={styles.GPage}>
      ...
);
```